### ISSUE TITLE
MultiPath point is loosing it's position after resize 

### ISSUE THREAD

**Issue by hasys** [ahome-it/lienzo-core#136]
After MultiPath resizing several times using different control points, at a given point one of the points gets crazy and the shape resizes in a bad way.

Steps to Reproduce:
1. Open "Wirez Resize" example under "Wires Examples" section in Lienzo Kitchen
2. Shift+Left Click on the shape
3. Move top right point to left bottom position
4. Repeat step 3 several times

Actual Result: 
![image](https://cloud.githubusercontent.com/assets/1477262/23063101/1992223c-f509-11e6-88d3-efd7ad70d8ab.png)

Expected Result:
![image](https://cloud.githubusercontent.com/assets/1477262/23063120/2b76e91a-f509-11e6-83cb-d51e01b4c5b9.png)

**Comment by helmutprovost** [comment]
Indeed, I could reproduce the very same issue on Google Chrome, Linux mint.
Repeating 2 times does not give any visual side effect.
